platform/ios/Mapbox-iOS-SDK.podspec

```diff
@@ -1,8 +1,8 @@
 Pod::Spec.new do |m|
 
-  version = '4.5.0'
+  version = '4.5.0-cn.1'
 
-  m.name    = 'Mapbox-iOS-SDK'
+  m.name    = 'Mapbox-iOS-SDK-cn'
```

Comment:
If the intention is to push this podspec to trunk, I suspect CocoaPods will complain if the name of this file does not match the name of the pod.